### PR TITLE
feat/new search engine ui

### PR DESCRIPTION
**Pull request by hemitoncode**

### CODE REVIEW

**Comment by hemitoncode** [comment]
please review this whenever you get the chance!

**Comment by cancelei** [comment]
@Hemit99123 I made adjustments in the search controller to default to values when there is no value passed to them. 

And tomorrow we meet, so I can help you set up your environment. (I updated the gem file to make it possible for Windows and Linux users to run the app smoothly).

So now we can search with the new UI.

I also found 2 issues:
The UI looks like this:
image

- The logo is missing, so I commented that line out for now.
- there is an issue with the display of different elements including low readability of general aspects of this specific page.

File: app/views/search/index.html.erb [hunk 1]
```diff
@@ -1,21 +1,71 @@
...
+            <circle cx="8.5" cy="8.5" r="7" />
+            <line x1="13.5" y1="13.5" x2="17.5" y2="17.5" />
+          </svg>
+        </div>
+    <%= form_with url: search_index_path, method: :get, local: true, class: "space-y-4" do |form| %>
+        <div class="pl-5 pr-5 sm:pl-0 sm:pr-0">
+          <%= form.text_field :query,  autocomplete: "off", type: "text", id: "searchBar", placeholder: "Enter search term", class: "block w-full mx-auto h-12 pl-10 pr-4 py-3 text-sm rounded-full bg-gray-300 placeholder-gray-500 focus:bg-gray-200 focus:outline-none focus:ring-2 focus:ring-[#f1f6e5]" %>
+        </div>
+          <%= form.hidden_field :search_engine, value: 'google' %> 
+          <%# default search engine results provider google, we should improve the logic later %>
```
Comment (cancelei) [inline app/views/search/index.html.erb:53]: > Changing this line would change the default search engine. It is hidden but passed to the controller when submit is pressed.

**Comment by hemitoncode** [comment]
ok got it. yea i thought there would be some ui issues. just help me set up tomorrow and we should be all fine. could you do around 11:00AM EST?

**Comment by cancelei** [comment]
> ok got it. yea i thought there would be some ui issues. just help me set up tomorrow and we should be all fine. could you do around 11:00AM EST?

I'm booked out for this time slot. Let's talk via discord about that.

**Comment by cancelei** [comment]
@Hemit99123 since we are migrating the Front End to a React Repo. I believe we should close this PR when time has come. So we optimize this repo for the backend activities.

**Comment by hemitoncode** [comment]
@cancelei yes I agree. This repo is now only for backend... Frontend stuff will be taken care of in ecosearch-frontend repository belonging to my github account

**Pull request closed without merging**